pkg/cli/admin/release/extract_tools.go

```diff
@@ -333,7 +333,19 @@ func (o *ExtractOptions) extractCommand(command string) error {
 		if currentOS == "*" {
 			currentArch = "*"
 		} else {
-			currentArch = "amd64"
+			parsed := strings.Split(currentOS, "/")
+			switch len(parsed) {
+			case 1:
+				// --command-os=linux
+				currentArch = "amd64"
```

Comment:
I think this is missing `currentOS = parsed[0]` ?

Reply:
It is set 3 line above